src/leap/common/events/signals.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# proto.py
+# Copyright (C) 2015 LEAP
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful",
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""
+Communication protocol definitions.
+"""
+
+
+from enum import Enum
```

Comment:
yes, I know it is used. but we're having troubles because (1) we have to backport it, (2) in some distros it might happen that some other package is requiring the installation of the old enum, which has a different api. all in all, I think we've spent more time dealing with packaging issues than the time it would have taken to code something simple. After all, we only need a simple class to wrap constants.